config/crds/starlingx_v1_system.yaml

```diff
@@ -297,15 +297,6 @@ spec:
                       name:
                         description: Name defines the system defined name of the filesystem
                           resource.
-                        enum:
-                        - cgcs
-                        - platform
-                        - cinder
-                        - database
-                        - docker-distribution
-                        - etcd
-                        - extension
-                        - patch-vault
```

Comment:
These currently line up with the set defined in gophercloud / controllerFilesystems.  Should we consider adjusting that list to only include the "well-known" file systems that need special handling?